auction-server/src/auction.rs

```diff
@@ -1428,10 +1546,26 @@ impl ChainStore for ChainStoreSvm {
             }])),
             None => {
                 // not yet confirmed
-                Ok(None)
+                if bids[0].initiation_time + BID_MAXIMUM_LIFE_TIME_SVM < OffsetDateTime::now_utc() {
+                    // If the bid is older than the maximum lifetime, it is expired even if it is submitted on chain
+                    Ok(Some(vec![BidStatusSvm::Expired { result: tx_hash }]))
+                } else {
+                    Ok(None)
+                }
             }
         }
     }
+
+    fn should_submit_auction(&self, permission_key: &Bytes) -> bool {
+        !permission_key.starts_with(&self.phantom_router_account.to_bytes())
```

Comment:
they could technically use different routers in the future, and for other similar style integrations, it may be annoying to add a bunch of these to the config. can we instead change the permission_keys iter to be a struct like

```
{
    permission_key: __,
    server_submit_auction: bool
}
```